src/main/kotlin/lottery/domain/Lotteries.kt

```diff
@@ -0,0 +1,15 @@
+package lottery.domain
+
+class Lotteries {
+    val lotteries = mutableListOf<Lottery>()
+
+    companion object {
+        fun makeAutoLotteries(number: Int): Lotteries {
+            val lotteries = Lotteries()
+            for (i in 0 until number) {
+                lotteries.lotteries.add(Lottery.makeAutoLottery())
+            }
+            return lotteries
+        }
+    }
+}
```

Comment:
객체를 생성할 때 가능하면 완벽한 상태로 생성하는게 좋습니다.
```suggestion
class Lotteries(
    val lotteries: List<Lottery>
) {

    companion object {
        fun makeAutoLotteries(number: Int): Lotteries {
            val lotteries = (0..number).map { Lottery.makeAutoLottery() }
            
            return Lotteries(lotteries)
        }
    }
}
```